src/middlewares/validateInputs.js

```diff
@@ -51,4 +51,4 @@ const updateFacility=(req,res,next)=>{
 }
 
 export {addAccomodation, addfacility,addFacilityComponent, 
-    updateFacilityComponent,updateAccomodation,updateFacility}
\ No newline at end of file
+    updateFacilityComponent,updateAccomodation,updateFacility}
```

Comment:
run linter/format your changes before commit

Reply:
Now it is fine no error